<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MediaPanel</class>
 <widget class="QWidget" name="MediaPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="Timeline" name="horizontalSlider">
       <property name="maximum">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="SpinBox" name="spinBox">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="accelerated">
        <bool>true</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="spacing">
      <number>24</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="SpinBox" name="spinBox_2">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="suffix">
          <string>F</string>
         </property>
         <property name="maximum">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="RangeSlider" name="horizontalSlider_2">
         <property name="maximum">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="SpinBox" name="spinBox_3">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="suffix">
          <string>F</string>
         </property>
         <property name="maximum">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>12</number>
       </property>
       <item>
        <widget class="MediaControl" name="start">
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="statusTip">
          <string>Go to start of animation [Shortcut F6]</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/animate/start</normaloff>:/animate/start</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MediaControl" name="previous">
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="statusTip">
          <string>Go to previous frame [Shortcut F]</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/animate/previous</normaloff>:/animate/previous</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MediaControl" name="reverse">
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="statusTip">
          <string>Play backwards</string>
         </property>
         <property name="styleSheet">
          <string notr="true">:!hover:!checked {
	background: #0066ff;
}</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/animate/reverse</normaloff>
           <normalon>:/animate/pause</normalon>:/animate/reverse</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MediaControl" name="play">
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="statusTip">
          <string>Play forwards [Shortcut F7]</string>
         </property>
         <property name="styleSheet">
          <string notr="true">:!hover:!checked {
	background: #0066ff;
}</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/animate/play</normaloff>
           <normalon>:/animate/pause</normalon>:/animate/play</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MediaControl" name="next">
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="statusTip">
          <string>Go to next frame [Shortcut G]</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/animate/next</normaloff>:/animate/next</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MediaControl" name="end">
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="statusTip">
          <string>Go to end of animation [Shortcut F8]</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/animate/end</normaloff>:/animate/end</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>12</number>
       </property>
       <item>
        <widget class="MediaControl" name="key">
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="statusTip">
          <string>Record Position, Rotation and PLA of Active Camera [Shortcut F9]</string>
         </property>
         <property name="styleSheet">
          <string notr="true">:!hover:!checked {
	background: #d40000;
}</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/animate/key</normaloff>:/animate/key</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="shortcut">
          <string>F9</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MediaControl" name="record">
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="statusTip">
          <string>Automatic Keyframing [Shortcut Ctrl+F9]</string>
         </property>
         <property name="styleSheet">
          <string notr="true">:!hover:!checked {
	background: #d40000;
}</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/animate/record</normaloff>:/animate/record</iconset>
         </property>
         <property name="shortcut">
          <string>Ctrl+F9</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MediaControl</class>
   <extends>QPushButton</extends>
   <header>mediacontrol.h</header>
  </customwidget>
  <customwidget>
   <class>SpinBox</class>
   <extends>QSpinBox</extends>
   <header>spinbox.h</header>
  </customwidget>
  <customwidget>
   <class>RangeSlider</class>
   <extends>QSlider</extends>
   <header>rangeslider.h</header>
   <slots>
    <signal>lowerBoundChanged(int)</signal>
    <signal>upperBoundChanged(int)</signal>
    <slot>setLowerBound(int)</slot>
    <slot>setUpperBound(int)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>Timeline</class>
   <extends>QSlider</extends>
   <header>timeline.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
